source/containers/auth/Settings/views/ImportAccountView/index.tsx

```diff
@@ -58,18 +58,24 @@ const ImportAccountView = () => {
       fileReader.readAsText(jsonFile, 'UTF-8');
       fileReader.onload = (ev: ProgressEvent<FileReader>) => {
         if (ev.target) {
+          const json = JSON.parse(ev.target.result as string);
```

Comment:
This should be wrapped in a try-catch block as it will throw an error if the string can not be parsed into a JSON object